helper/resource/testing_new_import_state.go

```diff
@@ -24,37 +25,25 @@ import (
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-func requirePlannableImport(t testing.T, versionUnderTest version.Version) error {
```

Comment:
This function has simply moved to the end of this file, along with other extracted helper functions.